File: 22.08/9.go
import (
    "fmt"
    "strconv"
)

func isPalindrome(x int) bool {
    tmp := strconv.Itoa(x)
    for i, v := range tmp {
        if byte(v) != tmp[len(tmp) - i - 1] {
            return false
        }
    }

    return true
}

File: typescript/23.09/9.ts
function isPalindrome(x: number): boolean {
    const str = String(x)
    let [i, j] = [0, str.length-1]
    while (i < j) {
        if (str[i++] !== str[j--]) {
            return false
        }
    }

    return true
};
